content/kubeone/master/guides/using_machine_deployment/_index.en.md

```diff
@@ -0,0 +1,19 @@
++++
+title = "Using Machine Deployments"
+date = 2021-04-05T12:00:00+02:00
+enableToc = true
++++
```

Comment:
This document seems to be similar to a document we already have: https://docs.kubermatic.com/kubeone/master/guides/machine_controller/

Can you extend the original document with the needed bits, so that we have only one document?

Reply:
ok let me try to merge the two

Reply:
I have merged this.. as requested. please check